opennms-webapp/src/main/webapp/admin/index.jsp

```diff
@@ -176,6 +176,7 @@
             <%String status = "Unknown";
                 try {
                     NotifdConfigFactory.init();
+                    NotifdConfigFactory.getInstance().reload();
                     status = NotifdConfigFactory.getPrettyStatus();
```

Comment:
same as above

Reply:
BTW have you startup a local instance and check the page working or not ?

Reply:
Verified.